### PR TITLE
Release v17.2.2

### PR DESCRIPTION
**Pull request by joaoandremartins**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -18,5 +18,5 @@ ideaEdition = IC
 ideaVersion = 2017.1
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.2.2_2017-SNAPSHOT
+version = 17.2.3
```
Comment (etanshaul) [inline gradle.properties]: see https://github.com/GoogleCloudPlatform/google-cloud-intellij/commit/e2696fac201422162ffd32b4b76d33de566271ae

the version should probably be 17.2.3_2017-SNAPSHOT

Comment (etanshaul) [inline gradle.properties]: looks like the tag is good, so you can just push a commit to this PR without bothering with another commit

Comment (joaoandremartins) [inline gradle.properties]: Didn't we decide that for the next version, master would start targeting 2017?

If so, 17.2.3 (even 17.3) should be the next version.

Comment (etanshaul) [inline gradle.properties]: yes but technically this is the 2017 branch so the next release from this branch would actually be _2017 if we happened to do it for whatever reason. If we target 2017 from master next, like you suggested, we'd be releasing from master.

Comment (joaoandremartins) [inline gradle.properties]: Cool, I can change it after this PR if needed

**Comment by patflynn** [comment]
I'm a bit confused what's going on.

We seem to have released two versions:
https://github.com/GoogleCloudPlatform/google-cloud-intellij/releases/

17.2.2 and 17.2.2_2017

17.2.2 is showing up as an update in my 2016.3 IJ release which I would not
expect since these are cut off the 2017 branch.

**Comment by joaoandremartins** [comment]
@patflynn I released 17.2.2 off of master, by mistake. Is there a way to remove it from the alpha channel?

**Comment by etanshaul** [comment]
what is 17.2.2 based off of? If its identical to the previous release in alpha e.g. 17.2.1 then its ok if its there.

**Comment by etanshaul** [comment]
also, you can delete the release from our github page. also you probably want to remove the tag too.

**Comment by patflynn** [comment]
Aha. No biggie. In that case like Etan mentioned we can just delete the
release.

**Comment by etanshaul** [comment]
I went ahead and deleted the accidental alpha release from JB (they provide a ui for it) since it was based of the latest tip in master which we don't want out there.

**Pull request merged**